package.json

```diff
@@ -190,7 +191,7 @@
     "build:js": "tsc --p tsconfig.build.json",
     "build:copy-assets": "cd src/lib && find . -name '*.scss' -o -name '*.css' -o -name '*.svg' | cpio -pdm ../../dist/",
     "build:rollup": "rollup -c && cp -r src/umd dist/",
-    "serve:docs": "DISABLE_ESLINT_PLUGIN=true npx styleguidist server --open",
+    "serve:docs": "DISABLE_ESLINT_PLUGIN=true FAST_REFRESH=false npx styleguidist server --open",
```

Comment:
Fixes an issue with the error screen in dev mode. If/when we upgrade to React 17, I think we can remove this.